bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
     capabilities: Seamless Upgrades
     categories: OpenShift Optional
     certified: "false"
-    createdAt: REPLACE_DATE
+    createdAt: "2023-05-22T16:06:59Z"
     description: An operator that enables Windows container workloads on OCP
-    olm.skipRange: '>=7.0.0 <8.0.0'
+    olm.skipRange: '>=8.0.0 <8.0.1'
```

Comment:
why bump the lower end of the skipRange here? what are the implications of this change?

Reply:
you mentioned [here](https://github.com/openshift/windows-machine-config-operator/pull/1566#discussion_r1176816796) that that's the expected functionality of the script, and these changes are from that script.

Reply:
imagine we were to run this script again with current functionality to prepare for version `8.0.2` -- the skipRange would be bumped to '>=9.0.0 <8.0.2' which doesn't look right
I suggest looking at [OLM upgrade process](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/how-to-update-operators.md), and how the skipRange looks in [other WMCO branches](https://github.com/openshift/windows-machine-config-operator/blob/release-4.11/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml)

Reply:
the way the bump works is that it takes the current version and puts it at the lower range, and then sets the updated version to the higher range. On an 8.0.2 bump, the skipRange would be >=8.0.1 <8.0.2
I can add a condition to only update that for the x version though. Should the lower range stay the same on minor versions?

Reply:
yes it should stay the same for minor versions, in short we do not need to touch skiprange for a minor version bump unless there is a need to do so.